lm75bd/lm75bd.c

```diff
@@ -25,9 +26,22 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
+#define CONF_WRITE_POINTER_BUFF_SIZE 1U
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  error_code_t errCode;
+  uint8_t buff[CONF_WRITE_POINTER_BUFF_SIZE] = {0};
+  errCode = i2cSendTo(devAddr, buff, CONF_WRITE_POINTER_BUFF_SIZE);
+  if (errCode != ERR_CODE_SUCCESS) return errCode;
+
+  uint8_t temp_bytes[2U] = {0}; // first interpret data as unsigned
```

Comment:
If you are going to define a macro for the length above, probably best to define a macro here as well